core/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -489,21 +490,60 @@ public int compare(Goal a, Goal b) {
                 }
               });
 
+          boolean inline = false;
+          String altText = " ";
+          int index = 1;
           for (Goal goal : sortedWools) {
             if (goal instanceof MonumentWool && goal.isVisible()) {
               MonumentWool wool = (MonumentWool) goal;
+              if (wool.getDefinition().isInline() == true) {
+                inline = true;
+              }
               if (!firstWool) {
-                woolText += "   ";
+                if (sortedWools.size() > 5) {
+                  woolText += " ";
+                  if (index % 2 == 0) {
```

Comment:
Adding a space every other wool looks extremely weird in the scoreboard, do either one or none, but don't toggle